Modules/_sqlite/row.c

```diff
@@ -166,8 +166,7 @@ pysqlite_row_subscript(PyObject *op, PyObject *idx)
             }
             if (eq) {
                 /* found item */
-                PyObject *item = PyTuple_GetItem(self->data, i);
-                return Py_XNewRef(item);
+                return PyTuple_GET_ITEM(self->data, i);
```

Comment:
```suggestion
                return Py_XNewRef(PyTuple_GET_ITEM(self->data, i));
```
?

Reply:
Yes, thanks!

Reply:
But, I'm pretty sure we can assert this item is not `NULL`.

Reply:
Addressed with 2ae9c189917b5cc0fd896c672997abec4f4bb7e7